pkg/k8s/labels.go

```diff
@@ -230,6 +230,14 @@ const (
 	// timeout in the proxy
 	ProxyInboundConnectTimeout = ProxyConfigAnnotationsPrefix + "/proxy-inbound-connect-timeout"
 
+	// ProxyOutboundDiscoveryCacheTimeout can be used to configure the timeout
+	// that will evict unused outbound discovery result
+	ProxyOutboundDiscoveryCacheTimeout = ProxyConfigAnnotationsPrefix + "/proxy-outbound-discovery-cache-unused-timeout"
+
+	// ProxyInboundDiscoveryCacheTimeout can be used to configure the timeout
+	// that will evict unused inbound discovery result
+	ProxyInboundDiscoveryCacheTimeout = ProxyConfigAnnotationsPrefix + "/proxy-inbound-discovery-cache-unused-timeout"
```

Comment:
nit, take it or leave it: should the names of the constants here reflect the actual name of the annotation?
```suggestion
	ProxyOutboundDiscoveryCacheTimeout = ProxyConfigAnnotationsPrefix + "/proxy-outbound-discovery-cache-unused-timeout"

	// ProxyInboundDiscoveryCacheUnusedTimeout can be used to configure the timeout
	// that will evict unused inbound discovery result
	ProxyInboundDiscoveryCacheUnusedTimeout = ProxyConfigAnnotationsPrefix + "/proxy-inbound-discovery-cache-unused-timeout"
```